extension_templates/annotation.py

```diff
@@ -179,3 +183,46 @@ def _update(self, X, Y=None):
 
         # implement here
         # IMPORTANT: avoid side effects to X, fh
+
+    # todo: set default parameters, so that a test instance can be created
+    @classmethod
+    def get_test_params(cls):
+        """Get default parameters of the estimator.
+
+        Returns
+        -------
+        params : dict or list of dict, default = {}
+            Default parameters related to the estimator class
+        """
+
+        # todo: set the default parameters for the estimators
+        # Default parameters can be dictionary or list of dictionaries
+        #
+        # example 1: specify params as dictionary
+        # any number of params can be specified
+        # params={"est":value0, "parama":value1, "paramb":value2}
+        #
+        # example 2: specify params as list of dictionary
+        # note: Only first dictionary will be used by create_test_instance
+        # list of dictionary can be of any length
+        # params=[{"parama":value1,paramb:value2},{"parama":value3,"paramb":value4}]
+        #
+        # This method would be called by create_test_instance.
```

Comment:
this is too verbose and duplicates content - can you shorten it?

Reply:
I have shortened this part, could you please review it?